File: STRINGS/1873/main.rb
# frozen_string_literal:true

wins = {
  'tesoura' => %w[papel lagarto],
  'papel' => %w[pedra spock],
  'pedra' => %w[lagarto tesoura],
  'lagarto' => %w[spock papel],
  'spock' => %w[tesoura pedra]
}

c = gets.to_i
c.times do
  rajesh, sheldon = gets.strip.split
  if wins[rajesh].include? sheldon
    puts 'rajesh'
  elsif wins[sheldon].include? rajesh
    puts 'sheldon'
  else
    puts 'empate'
  end
end

File: STRINGS/1873/main.py
wins = {
    'tesoura': ['papel', 'lagarto'],
    'papel': ['pedra', 'spock'],
    'pedra': ['lagarto', 'tesoura'],
    'lagarto': ['spock', 'papel'],
    'spock': ['tesoura', 'pedra']
}

c = int(input())
for _ in range(c):
    rajesh, sheldon = input().strip().split()
    if sheldon in wins[rajesh]:
        print('rajesh')
    elif rajesh in wins[sheldon]:
        print('sheldon')
    else:
        print('empate')
